package.json

```diff
@@ -10,8 +10,7 @@
     "prebuild": "npm run clean",
     "build": "gitbook build",
     "clean": "del _book",
-    "deploy": "npm run build && npm run deploy:rsync && git fetch",
```

Comment:
`npm run build` is executed by the tests. `deploy` needs only to `rsync` the produced artifacts (the `_book` directory).

This saves an execution of `npm run build`.

Now, if it's considered as an overkill optimization, or too unclear to be kept, I agree to build twice: a first time for tests (grepping warnings) and a second time for deployment.